score-web/src/app/business-term-management/business-term-create/business-term-create.component.html

```diff
@@ -12,7 +12,7 @@
 
         <mat-form-field>
           <mat-label>External Reference URI</mat-label>
-          <input matInput required [(ngModel)]="businessTerm.externalReferenceUri" maxlength="45"/>
+          <input matInput required [(ngModel)]="businessTerm.externalReferenceUri" maxlength="1000"/>
```

Comment:
We can allow it to the maximum length of the defined data type, `text`, which is 2^16-1 (=65535). Since the URI should contain only ASCII characters, we don't need to consider the problem caused by the size of unicode characters, but the process should verify the text whether it is the valid URI or not.

Reply:
just updated to 65535